### ISSUE TITLE
Gifts improved api request

### ISSUE THREAD

**Issue by CryoGenesisX** [BosslandGmbH/BuddyWing#33]
Can we get an improved api call for gifts (IsGifts)?
I'd like to check items in inventory by GiftType and GiftAffectionRank.
From my current logfile:
0x[4000000849010A75] itmGiftType Value: 1
0x[4000000849010A7A] itmGiftAffectionRank Value: 1
If i can get the Values of both i can create a more universal code for Gift usage and are not buggered with names, name changes and the likes.

**Comment by aevitas** [comment]
Definitely, should be trivial to add (it's likely already in the API but not publicly exposed)
